test/FunctionalTests/Microsoft.OData.Core.Tests/JsonLight/ODataJsonLightOutputContextTests.cs

```diff
@@ -749,7 +758,13 @@ public async Task WriteInStreamError_ThrowsExceptionIfInvokedWhenWritingBatchAsy
             Assert.Equal(Strings.ODataBatchWriter_CannotWriteInStreamErrorForBatch, exception.Message);
         }
 
-        #endregion Async Tests
+        [Fact]
+        public async Task DisposeAsync_ShouldDisposeStream_Writer_Asynchronously()

```

Comment:
What purpose does this empty test serve?

Reply:
My bad. Thought I had removed this.